server/HealthApp.sql

```diff
@@ -28,14 +28,34 @@ SET time_zone = "+00:00";
 --
 
 CREATE TABLE `users` (
-  `id` int(11) NOT NULL,
+  `id` int(11) NOT NULL PRIMARY KEY AUTO_INCREMENT,
   `firstname` varchar(512) NOT NULL,
   `surname` varchar(512) NOT NULL,
   `email` varchar(512) NOT NULL,
   `password` varchar(512) NOT NULL,
   `role` varchar(10) NOT NULL DEFAULT 'user'
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 
+
+CREATE TABLE `profile` (
+  `profileId` int(11) NOT NULL PRIMARY KEY AUTO_INCREMENT, 
+  `Firstname` varchar(512) NOT NULL,
+  `Surname` varchar(512) NOT NULL,
+  `Email` varchar(512) NOT NULL,
+  `Age` varchar(512),
```

Comment:
Some of these fields are already existing in the users page. You can pass them to the profile component from App.jsx theres no need to set them again. Take a look at App.jsx line 18 where these fields are defined and then populated in the useEffect (line 30).